pkg/hintrunner/zero/zerohint_others.go

```diff
@@ -751,3 +751,50 @@ func createNormalizeAddressHinter(resolver hintReferenceResolver) (hinter.Hinter
 
 	return newNormalizeAddressHint(isSmall, addr), nil
 }
+
+// Sha256AndBlake2sInput hint writes 1 or 0 at `full_word` address, wether `n_bytes“
+// is greater than or equal to 4 or not
+//
+// `newSha256AndBlake2sInputHint` takes 2 arguments
+//   - `full_word` represents the address where the result of the comparison is stored
+//   - `n_bytes` represents the value that will be compared to 4
+func newSha256AndBlake2sInputHint(fullWord, nbytes hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "Sha256AndBlake2sInput",
```

Comment:
i would add a quick unit test for this too

Reply:
Sure

Reply:
done!